messaging-activemq/subsystem/src/test/java/org/wildfly/extension/messaging/activemq/MessagingActiveMQSubsystem_17_0_TestCase.java

```diff
@@ -0,0 +1,238 @@
+        ModelNode serverModel = rootModel.require(SUBSYSTEM).require(MessagingExtension.SUBSYSTEM_NAME).require(SERVER)
+                .require(DEFAULT);
+
+        Assert.assertEquals(1357, serverModel.get(ServerDefinition.JOURNAL_BUFFER_TIMEOUT.getName()).resolve().asInt());
+        Assert.assertEquals(102400, serverModel.get(ServerDefinition.JOURNAL_FILE_SIZE.getName()).resolve().asInt());
+        Assert.assertEquals(2, serverModel.get(ServerDefinition.JOURNAL_MIN_FILES.getName()).resolve().asInt());
+        Assert.assertEquals(5, serverModel.get(ServerDefinition.JOURNAL_POOL_FILES.getName()).resolve().asInt());
+        Assert.assertEquals(7, serverModel.get(ServerDefinition.JOURNAL_FILE_OPEN_TIMEOUT.getName()).resolve().asInt());
+        kernelServices.shutdown();
+    }
+
+    @Test
+    public void testBridgeCallTimeout() throws Exception {
+        KernelServices kernelServices = standardSubsystemTest(null, false);
+        ModelNode rootModel = kernelServices.readWholeModel();
+        ModelNode bridgeModel = rootModel.require(SUBSYSTEM).require(MessagingExtension.SUBSYSTEM_NAME).require(SERVER)
+                .require(DEFAULT).require(BRIDGE).require("bridge1");
+
+        Assert.assertEquals("${call.timeout:60000}", bridgeModel.get(BridgeDefinition.CALL_TIMEOUT.getName()).asExpression().getExpressionString());
+        Assert.assertEquals(60000, bridgeModel.get(BridgeDefinition.CALL_TIMEOUT.getName()).resolve().asLong());
+        kernelServices.shutdown();
+    }
+
+    /////////////////////////////////////////
+    //  Tests for HA Policy Configuration  //
+    /////////////////////////////////////////
+    @Test
+    public void testHAPolicyConfiguration() throws Exception {
+        standardSubsystemTest("subsystem_17_0_ha-policy.xml");
+    }
+
+    ///////////////////////
+    // Transformers test //
+    ///////////////////////
+    @Test
+    public void testTransformersWildfly35() throws Exception {
+        testTransformers(ModelTestControllerVersion.MASTER, MessagingExtension.VERSION_16_0_0);
+    }
+    @Test
+    public void testTransformersWildfly28() throws Exception {
+        testTransformers(ModelTestControllerVersion.MASTER, MessagingExtension.VERSION_15_0_0);
+    }
+
+    @Test
+    public void testTransformersWildfly27() throws Exception {
+        testTransformers(ModelTestControllerVersion.MASTER, MessagingExtension.VERSION_14_0_0);
+    }
+
+    @Test
+    public void testTransformersWildfly26_1() throws Exception {
+        testTransformers(ModelTestControllerVersion.MASTER, MessagingExtension.VERSION_13_1_0);
+    }
+
+    @Test
+    public void testTransformersWildfly25() throws Exception {
+        testTransformers(ModelTestControllerVersion.MASTER, MessagingExtension.VERSION_13_0_0);
+    }
+
+    @Test
+    public void testTransformersEAP_7_4_0() throws Exception {
+        testTransformers(EAP_7_4_0, MessagingExtension.VERSION_13_0_0);
+    }
```

Comment:
You will also need to test the transformation against EAP_8_0_0, and, once https://github.com/wildfly/wildfly-legacy-test/pull/40 gets merged and https://issues.redhat.com/browse/WFCORE-7254 gets resolved, against EAP_8_1_0

Reply:
@ehsavoie This is still an open question; you have not added any transformation against EAP 8.1, so can we assume you prefer to leave it as it is?